colors/nord.vim

```diff
@@ -234,7 +234,7 @@ call s:hi("Directory", s:nord8_gui, "", s:nord8_term, "NONE", "", "")
 call s:hi("EndOfBuffer", s:nord1_gui, "", s:nord1_term, "NONE", "", "")
 call s:hi("ErrorMsg", s:nord4_gui, s:nord11_gui, "NONE", s:nord11_term, "", "")
 call s:hi("ModeMsg", s:nord4_gui, "", "", "", "", "")
-call s:hi("MoreMsg", s:nord4_gui, "", "", "", "", "")
+call s:hi("MoreMsg", s:nord14_gui, "", "", "", "", "")
```

Comment:
This is not just a compatibility change but a different color for the group at all. Could you please post comparison screenshots between Vim in _terminal_ and _GUI_ mode?

Reply:
This is the comparison of highlighting for the `MoreMsg` group between console vim (`set notermguicolors`) and gvim prior to this PR:
gvim:
![Screenshot from 2020-04-28 09:03:12](https://user-images.githubusercontent.com/16892446/80490765-d77d4200-892f-11ea-8418-96c59c81b378.png)
vim:
![Screenshot from 2020-04-28 09:05:17](https://user-images.githubusercontent.com/16892446/80490809-e6fc8b00-892f-11ea-818e-de5c0a757a40.png)

This patch changes the highlighting for `gvim` to match that of console `vim`:
gvim after the patch:
![Screenshot from 2020-04-28 09:10:31](https://user-images.githubusercontent.com/16892446/80491100-422e7d80-8930-11ea-9a87-df3121e13899.png)

Reply:
Ah, I've checked `:help more-prompt` (`:help MoreMsg` links to it) and know I understand why you've used `nord14`. Since there's no color defined for _terminal_ mode it uses the default color which is some kind of green.
I've read `:more` and tested the actual prompt by running

```vim
echon "This is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\nThis is a message test for arcticicestudio/nord-vim#202\n"
```

To ensure it matches _Nord_'s style we should change this to use `nord8` (main accent color) for both _terminal_ and _GUI_ mode.

<p align="center"><em>Vim</em>&apos;s default style</p>
<div><img src="https://user-images.githubusercontent.com/7836623/80496836-01ebf100-896a-11ea-8a56-a1cff80adfd8.png" /></div>

<p align="center">Using <code>nord8</code> for both <em>terminal</em> and <em>GUI</em> mode</p>
<div><img src="https://user-images.githubusercontent.com/7836623/80496833-00222d80-896a-11ea-82f8-f7636551b687.png" /></div>

```suggestion
call s:hi("MoreMsg", s:nord8_gui, "", s:nord8_term, "", "", "")
```

Reply:
Yeah definitely. I think `nord8` definitely makes more sense. And I am happy that now the `terminal` mode and `gui` mode uses the same color for `MoreMsg`.